### ISSUE TITLE
Requesting new Provider: Orthopedic + Fracture Specialists

### ISSUE THREAD

**Issue by joestump** [fastenhealth/fasten-onprem#335]
Not entirely clear to me who the patient portal provider is. I think it's a JP Morgan company.

**Comment by AnalogJ** [comment]
Can you confirm that this is the correct patient url? https://www.orthopedicandfracturespecialists.com/

**Comment by joestump** [comment]
Yes, that’s correct. I couldn’t find the underlying platform provider but I think I saw a reference to JP Morgan.

**Comment by AnalogJ** [comment]
Hey @joestump sorry for the delay looking into this. 
To be honest, I still dont know who the developer is behind *.myezyaccess.com patient portals. 

I think its these guys - https://www.ezaccessmot.com/ 

```
* ALPN, server did not agree to a protocol
* Server certificate:
*  subject: C=US; ST=Tennessee; O=Medical Office Technologies, Inc.; CN=*.myezyaccess.com
*  start date: Oct 22 00:00:00 2023 GMT
*  expire date: Nov 21 23:59:59 2024 GMT
*  subjectAltName: host "oandfs.myezyaccess.com" matched cert's "*.myezyaccess.com"
*  issuer: C=US; O=Network Solutions L.L.C.; CN=Network Solutions RSA OV SSL CA 3
```

**Comment by AnalogJ** [comment]
they say `ezAccess relies on athenahealth’s data export ability for 170.315 (b)(10). [Click Here](https://docs.mydata.athenahealth.com/fhir-r4/ehiexport.html#electronic-health-information-ehi-export) for documentation. Both single patient and population export is supported.` which is kind of confusing as wel.
